src/app/pages/race/race.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="breed-container">
+  <section class="breed-introduction">
+    <h2 class="title-introduction">Résumé des races</h2>
+    <p>
+      L’univers d’ Expenssion-One est basé sur quatre races primaires et six
+      races secondaires. Lors de la création du monde, seules quatre races
+      furent créées par les dieux d’Astris.  <br />Les Humains représentaient la
+      majorité et devaient faire prospérer la terre ferme. <br />Les Zargas
+      devaient protéger les fonds marins ainsi que leurs habitants, tandis que
+      les Gobelins devaient eux protéger les grottes et fractures terrestres. 
+      <br />Les Anges devaient eux protéger les cieux et les îles volantes. Le
+      rôle des gobelins était primordial, car les sols des royaumes contiennent
+      bon nombre de ressources dont, le Deraris, un minerais magique condensé
+      hautement radioactif. <br />Mais les Humains, ont fait preuve d'intérêt
+      sans borne pour la puissance du Déraris, et son extraction abusive a alors
+      provoqué des transformations physiques chez plusieurs d’entre-eux.  En
+      découla des distinctions raciales dûes à de nouveaux traits physiques; les
+      races secondaires étaient nées.
+    </p>
+  </section>
+
+  @for (race of races; track race) {
+  <div class="list-breed-container">
```

Comment:
li instead of div since you'll have a list oh races

Reply:
I changed the class name for more sense